.github/workflows/devtools.yml

```diff
@@ -162,10 +162,14 @@ jobs:
       working-directory: ${{ github.workspace }}/lighthouse
     - name: yarn smoke --runner devtools
       # Disabled because normal Chrome usage makes DevTools not function on these poorly constructed pages
-      #   errors-expired-ssl errors-infinite-loop
+      #   errors-expired-ssl errors-infinite-loop dbw
```

Comment:
whats the problem with the dbw page?

Reply:
I struggled to run Lighthouse from DevTools on the page unless I dismissed the prompt that comes up.

Reply:
The comment above is pretty self-evident for the first two cases here, but not for dbw. can you add `dbw (dialog prompt)`